iphone/iphone/Titanium.xcodeproj/project.pbxproj

```diff
@@ -3002,6 +3003,7 @@
 					"\"$(SRCROOT)/../Classes/APSAnalytics\"",
 					"\"$(SRCROOT)/../Classes/APSHTTPClient\"",
 				);
+				OTHER_CFLAGS = "-DNDEBUG";
```

Comment:
@cb1kenobi Question here: Can I do it this way or should I add it via the _build.js? I am not sure if our vanilla project may strip these settings.